linode_api4/objects/base.py

```diff
@@ -210,9 +210,29 @@ def save(self, force=True) -> bool:
         if not force and not self._changed:
             return False
 
-        resp = self._client.put(
-            type(self).api_endpoint, model=self, data=self._serialize()
-        )
+        if not self._populated:
+            result = {
+                a: getattr(self, a)
+                for a in type(self).properties
+                if type(self).properties[a].mutable
+                and object.__getattribute__(self, a) is not None
+                and hasattr(self, a)
```

Comment:
I think we can remove `hasattr(...)` since we're checking for whether the key exists using `object.__getattribute__(...)`.

Reply:
Fixed!